### PR TITLE
Fix all linter issues with the new golangci-lint config

### LINKED ISSUE #103: Feature: improve code style
**Is your feature request related to a problem? Please describe.**
File.d codebase contains other code styles.

**Describe the solution you'd like**
Configure the golangci-lint and fix common styling issues.

### PR DESCRIPTION
**Pull request by vadimalekseev**
Fixes #103

### CODE REVIEW

File: cfg/config.go [hunk 1]
```diff
@@ -15,6 +14,10 @@ import (
 	"github.com/ozontech/file.d/logger"
 )
 
+const (
+	trueStr = "true"
```
Comment (vadimalekseev) [inline cfg/config.go:18]: use vars for repeatable values

File: cfg/config.go [hunk 2]
```diff
@@ -53,7 +56,7 @@ func NewConfig() *Config {
 
 func NewConfigFromFile(path string) *Config {
 	logger.Infof("reading config %q", path)
-	yamlContents, err := ioutil.ReadFile(path)
+	yamlContents, err := os.ReadFile(path)
```
Comment (vadimalekseev) [inline cfg/config.go:59]: ioutil is deprecated

File: cfg/config.go [hunk 3]
```diff
@@ -129,13 +132,13 @@ func parseConfig(json *simplejson.Json) *Config {
 	}
 	config.Vault.ShouldUse = config.Vault.Address != "" && config.Vault.Token != ""
 
-	pipelinesJson := json.Get("pipelines")
-	pipelines := pipelinesJson.MustMap()
+	pipelinesJSON := json.Get("pipelines")
```
Comment (vadimalekseev) [inline cfg/config.go:135]: Capitalize "JSON"

File: decoder/consts.go [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+	NO Type = iota
 	AUTO
 	JSON
 	RAW
 	CRI
 	POSTGRES
 )
 
-type DecoderType int
+type Type int
```
Comment (vadimalekseev) [inline decoder/consts.go:12]: Use decoder.Type instead of decoder.DecoderType

File: metric/metrics.go [hunk 5]
```diff
@@ -16,7 +16,7 @@ const (
 
 type metricType = int
 
-type MetricDesc struct {
+type Desc struct {
```
Comment (vadimalekseev) [inline metric/metrics.go:19]: Use metric.Desc instead of metric.MetricDesc

File: pipeline/batch.go [hunk 6]
```diff
@@ -139,7 +136,7 @@ func (b *Batcher) work(ctx context.Context) {
 	}
 }
 
-func (b *Batcher) commitBatch(ctx context.Context, events []*Event, batch *Batch) []*Event {
+func (b *Batcher) commitBatch(events []*Event, batch *Batch) []*Event {
```
Comment (vadimalekseev) [inline pipeline/batch.go:139]: context is unused

File: pipeline/processor.go [hunk 7]
```diff
@@ -68,7 +68,7 @@ type processor struct {
 
 var id = 0
 
-func NewProcessor(
+func newProcessor(
```
Comment (vadimalekseev) [inline pipeline/processor.go:71]: NewProcessor is only used in the current package

File: pipeline/processor.go [hunk 8]
```diff
@@ -146,23 +145,23 @@ func (p *processor) processSequence(event *Event) bool {
 		p.output.Out(event)
 	}
 
-	return isSuccess
+	return true
 }
 
-func (p *processor) processEvent(event *Event) (isSuccess bool, isPassed bool, e *Event) {
+func (p *processor) processEvent(event *Event) (isPassed bool, e *Event) {
```
Comment (vadimalekseev) [inline pipeline/processor.go:151]: isSuccess is unused

File: plugin/README.md [hunk 9]
```diff
@@ -387,7 +387,8 @@ If appears we try to send event to this bucket instead of described here.
 > ⚠ Currently bucket names for bucket and multi_buckets can't intersect.
 
 > ⚠ If dynamic bucket moved to config it can leave some not send data behind.
-> To send this data to s3 move bucket dir from /var/log/dynamic_buckets/bucketName to /var/log/static_buckets/bucketName (/var/log is default path)
+> To send this data to s3 move bucket dir from /var/log/dynamic_buckets/bucketName to
```
Comment (vadimalekseev) [inline plugin/README.md:390]: `make gen-doc` changes

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -55,6 +55,12 @@ git push origin <topic-branch-name>
 ```
 7. Open a pull request with a clear title and description.
 
+## Code style
+
+* log in lower case without a dot in the end of a sentence;
```
Comment (vitkovskii) [inline CONTRIBUTING.md:60]: Documentation sentence should start with uppercase letter here

Comment (vitkovskii) [reply CONTRIBUTING.md:60]: And also, let's remove `;` at the end

**Comment by vadimalekseev** [comment]
now it is not relevant

**Pull request closed without merging**